pkg/edit/editor.go

```diff
@@ -47,17 +50,18 @@ func NewEditor(tty cli.TTY, ev *eval.Evaler, st store.Store) *Editor {
 	}
 
 	if fuser != nil {
+		filters := newListVar(vals.EmptyList)
+		ed.ns["add-cmd-filters"] = filters
+
 		appSpec.AfterReadline = []func(string){func(code string) {
-			if code != "" && !strings.HasPrefix(code, " ") {
+			if code != "" && !strings.HasPrefix(code, " ") &&
```

Comment:
Now that we have this general filtering mechanism, can you implement the default filter (skipping commands that start with a space) as a *default value* of the filter list?

You can define a simple builtin function as the default element: `func(s string) bool { return !strings.HasPrefix(s, " ") }`